src/backend/preprocessor/cfgGenerator/FunctionCall.js

```diff
@@ -0,0 +1,47 @@
+define([
+    '../Cfg'
+], function (Cfg) {
+    var cfgGenerator;
+
+    function FunctionCall (paramNode, options) {
+	var functionName = paramNode.name;
+	var isVariadic = options && options.prototypes[functionName].isVariadic;
+	var parameters = paramNode.parameters;
+
+	var resolveInstr = new Cfg({
+	    type: 'RESOLVE',
+	    param: functionName
+	});
+	var result;
+
+	if(parameters && parameters.length > 0) {
+	    result = cfgGenerator(parameters[0], options); 
```

Comment:
please try to introduce helper vars for things like parameters[0], for clarity. You don't have to fix everything, just a tip for future